backlog/src/Bucket.cs

```diff
@@ -0,0 +1,38 @@
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+using Amazon.S3;
+using Amazon.S3.Model;
+
+namespace Backlog.Src
+{
+
+    class Bucket
+    {
+
+        // private static readonly Amazon.RegionEndpoint London = Amazon.RegionEndpoint.EUWest2;
+
+        private static readonly string BucketName = Environment.GetEnvironmentVariable("BUCKET_NAME");
+
+        // private static readonly string AccessKeyId = Environment.GetEnvironmentVariable("ACCESS_KEY_ID");
```

Comment:
@jurisdatum we should remove these comented strings right? I ran batch 3 without these as the sdk will read first fro a config file and then fro the env vars in the terminal